tests/data/edlifo-project/Exports/s0/ExportsActual.toml

```diff
@@ -0,0 +1,135 @@
+[[Loads]]
```

Comment:
Any file in the Exports directory should NOT be part of the repository. Only input files should be present.

Reply:
Okay